### DISCUSSION TITLE
Traducción de flutter sin el codemagic yaml

### DISCUSSION THREAD

**Discussion by lualbertovargas** [codemagic-ci-cd/codemagic-docs#2621] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hacer todo el CI CD y esperar

### Expected results

Error en traducción con el work flow propuesto

### Actual results

Error

**Comment by lualbertovargas** [comment]
Dice no se encuentra app localizations https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization

**Reply by dtrdic** [reply]
hey @lualbertovargas can you please give me more information of the issue you are having? Please share a build id as well, it can be found on the build overview page. Let's please use English for the rest of the conversation as my Spanish is not very good:)

**Comment by icarusdust** [comment]
Closing the thread due to inactivity. You can re-open it by replying to it.
